index.html

```diff
@@ -73,12 +74,12 @@ <h3 class="panel-title">Day</h3>
         </div>
       </div>
 
-      <div class="panel panel-danger" id="block-panel">
+      <div class="panel panel-danger">
         <div class="panel-heading">
-          <h3 class="panel-title">Current Block</h3>
+          <h3 class="panel-title" id="Derrick">Until the End of Block</h3>
```

Comment:
Also, 'Until the End of Block' is a bit long and wraps on mobile screens, can you make it a few less words?